src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.ts

```diff
@@ -26,8 +29,10 @@ export class ExamNavigationBarComponent implements OnInit {
     criticalTime = moment.duration(5, 'minutes');
 
     icon: IconProp;
+    getSubmissionForExercise = this.examParticipationService.getSubmissionForExercise;
```

Comment:
Are you sure this is good practice to store functions like this? Why not just call the function directly when you need it?

Reply:
it makes the rest of the code easier readable imo, but I removed it nonetheless 👍